drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -360,18 +361,30 @@ function DragAndDropTemplates(configuration) {
     var sidebarTemplate = function(ctx) {
         var showAnswerButton = null;
         if (ctx.show_show_answer) {
+            var options = {
+                disabled: ctx.showing_answer ? true : ctx.disable_show_answer_button,
+                spinner: ctx.show_answer_spinner
+            };
             showAnswerButton = sidebarButtonTemplate(
                 "show-answer-button",
                 "fa-info-circle",
                 gettext('Show Answer'),
-                ctx.showing_answer ? true : ctx.disable_show_answer_button,
-                ctx.show_answer_spinner
+                options
             );
         }
+        var go_to_beginning_button_class = 'go-to-beginning-button';
+        if (!ctx.show_go_to_beginning_button) {
+            go_to_beginning_button_class += ' sr';
+        }
         return(
             h("section.action-toolbar-item.sidebar-buttons", {}, [
+                sidebarButtonTemplate(go_to_beginning_button_class, "fa-arrow-up", gettext("Go to Beginning"), {
```

Comment:
Nit: Since this has to cross multiple lines anyway, I'd like to see one argument per line. Makes it easier to diff later on.